plugin.video.lacapitv/resources/settings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
This uses the old format for addon settings, can you convert this to the new format? see https://forum.kodi.tv/showthread.php?tid=350135

Reply:
I don't understand is changing utf by UTF?

Reply:
No, the XML schema is completely different. In the thread I linked there is an add-on you can use to convert them automatically on your behalf.